dolphin/cmd/api.py

```diff
@@ -42,6 +58,10 @@ def main():
     log.setup(CONF, "dolphin")
     utils.monkey_patch()
 
+    if is_port_in_use('localhost', CONF.dolphin_listen_port):
```

Comment:
May not be 'localhost' always, can get from config

Reply:
done